elytron/src/main/java/org/wildfly/extension/elytron/ElytronExtension.java

```diff
@@ -75,6 +75,7 @@ public class ElytronExtension implements Extension {
     static final ModelVersion ELYTRON_16_0_0 = ModelVersion.create(16);
     static final ModelVersion ELYTRON_17_0_0 = ModelVersion.create(17);
     static final ModelVersion ELYTRON_18_0_0 = ModelVersion.create(18);
+    static final ModelVersion ELYTRON_18_1_0 = ModelVersion.create(18, 1);
```

Comment:
The model itself should just bump to 20, the subsystem only registers one version at a time so would never register as 18.1